### DISCUSSION TITLE
Yahoo.com, Hotmail and Other Domains Not Getting Verification Email

### DISCUSSION THREAD

**Discussion by metasoul111** [Web3Auth/Web3Auth#1156] [Help]
Our end users with Yahoo.com, Hotmail, Outlook and some others like t-online.de addresses are not getting the verification email to login. Richard from your support said yesterday that this was fixed but we are still having issues. Can anyone assist?

**Comment by tanguyenvn** [comment]
Could you send us email addresses to check? Normally, when users click unsubscribe from mail list, then they won't be able to receive emails. If that is the cause then you can ask your users to re-subscribe to our mailing list. e.g. open old email and change mail subscription in there

**Reply by metasoul111** [reply]
@tainguyentt there are several, but due to our privacy policy I cannot share the email with third parties. the domains are t-online.de , earthlink.com

**Reply by metasoul111** [reply]
@chaitanyapotti @tainguyentt is there any update on this issue? Thank you.

**Comment by metasoul111** [comment]
@chaitanyapotti @tainguyentt bumping this again, is there any updates on this issue?

**Comment by metasoul111** [comment]
@chaitanyapotti @tainguyentt we are now having issues with aol.com emails, the verification emails are not arriving. This is becoming a big issue, any updates?

**Comment by chaitanyapotti** [comment]
Hello..we're using sendgrid service and these domains sometimes block our sender id.
we're in the process of migrating to aws ses to send emails
